recipes/libgit2/pstdint.h

```diff
@@ -0,0 +1,912 @@
+/*  A portable stdint.h
```

Comment:
So this file shouldn't be needed. Just add `msinttypes` to your build dependencies (for Windows only) and remove this.